components/applications-service/pkg/storage/postgres/services.go

```diff
@@ -242,15 +248,18 @@ func (db *Postgres) GetServicesDistinctValues(fieldName, queryFragment string) (
 	}
 
 	columnName := columnNameForField(fieldName)
-	query := fmt.Sprintf("SELECT DISTINCT %s from service_full AS t WHERE t.%s ILIKE $1 ORDER BY %s ASC LIMIT 100;",
+	queryFirst := fmt.Sprintf("SELECT DISTINCT %s from service_full AS t WHERE t.%s ILIKE $1 ",
 		columnName,
```

Comment:
I know we pre-validate the values of fieldName elsewhere in the code, but might it be worth also quoting some of these identifiers as a second line of defense? We can quote them with a function like: https://github.com/chef/automate/blob/a4ba9f1f2f9b860fefc40c1b85e05849f61b52bd/lib/platform/pg/queries.go#L117-L129 that might be appropriate to move somewhere shared. #

Reply:
Also, one trick you can do to avoid passing multiple times the same parameter and reuse:
```go
queryFirst := fmt.Sprintf("SELECT DISTINCT %[1]s from service_full AS t WHERE t.%[1]s ILIKE $1 ", columnName)
```

_Make sure to use the indicator `%[1]s`_